terraform-organize-configuration/step1.md

```diff
@@ -0,0 +1,65 @@
+In this step, you will create two S3 buckets configured for static website
+hosting using a monolithic configuration.
+
+## Review AWS Provider
+
+Open `main.tf`{{open}}. Your configuration begins with the AWS provider block
+below.
+
+```
+provider "aws" {
+  region = var.aws_region
+}
+```
+
+This Katacoda scenario uses [localstack](https://localstack.cloud/) to provide a 
+testing environment on your local machine that provides the same functionality 
+and APIs as the real AWS cloud environment. This way you can learn how to organize
+your Terraform configuration without needing an AWS account.
```

Comment:
Same as above. It's unclear what "local" means here because it's on Katacoda rather than the student's actual machine.